style.css

```diff
@@ -1,75 +1,87 @@
-*{
-    margin: 0;
-    padding: 0;
-    box-sizing: border-box;
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
 }
-body{
-    background:linear-gradient( rgba(78, 151, 206, 0.5) 100%
-    , rgba(0, 0, 0, 0.5)100%),url("https://assets.weforum.org/article/image/responsive_big_webp_LRQb39ZdkYto2vRkr0IpUGi9-5Yvx_plqhjGDAzBxYo.webp");  
-text-align: center;
+
+body {
+  background:
+    linear-gradient(
+      rgba(78, 151, 206, 0.5) 100%,
+      rgba(0, 0, 0, 0.5) 100%
+    ),
+    url("https://assets.weforum.org/article/image/responsive_big_webp_LRQb39ZdkYto2vRkr0IpUGi9-5Yvx_plqhjGDAzBxYo.webp");
+  text-align: center;
 }
-.main{
-    text-align: center; 
-    color: white;
-    font-size: 18px;
-    padding: 30px 0px;
+
+.main {
+  text-align: center;
+  color: white;
+  font-size: 18px;
+  padding: 30px 0;
 }
 
 #title {
-margin: 10px 0px;
+  margin: 10px 0;
 }
 
 #description {
-    margin-bottom: 30px;
-    font-size: 23px;
+  margin-bottom: 30px;
+  font-size: 23px;
 }
 
 #survey {
-    background-color: hsl(240, 100%, 10%);
-    /* color: white; */
-    margin: 10px auto;
-    padding: 30px 40px;
-    border-radius: 10px;
-    height: auto;
-    text-align: justify;
-    width: 50%;
-}
+  background-color: hsl(240, 87%, 15%);
 
+  /* color: white; */
+  margin: 10px auto;
+  padding: 30px 40px;
+  border-radius: 10px;
+  height: auto;
+  text-align: justify;
+  width: 50%;
+}
 
-#name, #email, #number, #drop, #dropdown {
-    width: 100%;
-    height: 50%;
-    padding: 10px;
-    border-radius: 7px;
-    border: none;
-    margin: 8px 0px 30px 0px;
+#name,
+#email,
+#number,
+#drop,
+#dropdown {
+  width: 100%;
+  height: 50%;
+  padding: 10px;
+  border-radius: 7px;
+  border: none;
+  margin: 8px 0 30px 0;

```

Comment:
![Submit](https://user-images.githubusercontent.com/123330794/236797785-a1345246-72ac-4577-9646-71dcd4f43c48.PNG)

Reply:
- [ ] Please consider adjusting the alignment of your button by removing some of the empty space at the bottom of your card. This can help create a more visually pleasing and balanced design, making the button look more integrated with the rest of the card.

Reply:
-  [ ] Good clear comment with reason for change 👏🏾